src/wrap-store/wrapStore.js

```diff
@@ -99,7 +99,9 @@ export default (store, {
   /**
    * Setup external action handler
    */
-  chrome.runtime.onMessageExternal.addListener(dispatchResponse);
+  if (chrome.runtime.onMessageExternal) {
```

Comment:
Does `chrome.runtime` exist in FF? I'm assuming it does, but maybe we should create a utility file called `features` that has functions like:

```
export const hasMessageExternal = () => {
  return chrome && chrome.runtime && chrome.runtime.onMessageExternal;
};
```

That way, when we want to support other browsers (like IE Edge), we can simply change the logic in the functions once and it should largely apply across the libray.